google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -229,9 +229,20 @@ public boolean isRefreshingChannel() {
 
   /** Returns a builder for the default ChannelProvider for this service. */
   public static InstantiatingGrpcChannelProvider.Builder defaultGrpcTransportProviderBuilder() {
-    return BigtableStubSettings.defaultGrpcTransportProviderBuilder()
+    InstantiatingGrpcChannelProvider.Builder builder =
+        BigtableStubSettings.defaultGrpcTransportProviderBuilder()
         .setPoolSize(getDefaultChannelPoolSize())
         .setMaxInboundMessageSize(MAX_MESSAGE_SIZE);
+    // TODO(weiranf): Remove this once DirectPath goes to public beta
+    if (isDirectPathEnabled()) {
+      builder.setAttemptDirectPath(true);
+    }
+    return builder;
+  }
+
+  // TODO(weiranf): Remove this once DirectPath goes to public beta
+  private static boolean isDirectPathEnabled() {
+    return Boolean.getBoolean("bigtable.attempt-directpath");
```

Comment:
Please cleanup the constants as well:
- remove DIRECT_PATH_ENV_VAR
- update comment for DIRECT_PATH_ENDPOINT